amigo_skills/test/test_amigo.test

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" ?>
+<launch>
+    <!-- N.B.: test-name MUST be amigo in order for namespacing to work -->
```

Comment:
This kind of assumptions should be documented in `robot_skills`

Reply:
Good point, no need for people to reinvent the wheel. See f96f2719fbde0d56b7e0774b22becbe009a44bd3 (was the best location I could think of)